doc/source/SSHClient.rst

```diff
@@ -107,9 +107,11 @@ API: SSHClient and SSHAuth.
         :type open_stdout: bool
         :param open_stderr: open STDERR stream for read
         :type open_stderr: bool
+        :param stdin: pass STDIN text to the process
+        :type stdin: ``typing.Optional[str]``
```

Comment:
`typing.Optional[bytes]` and need to check for `bytearray` support (both Subprocess and SSH)